bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/internal/ImageList.java

```diff
@@ -445,4 +457,22 @@ public int size () {
 	return result;
 }
 
+public static long win32_getHandle(ImageList imageList, int zoom) {
+	if(!imageList.zoomToHandle.containsKey(zoom)) {
+		int scaledWidth = DPIUtil.scaleUp(DPIUtil.scaleDown( imageList.width,imageList.zoom), zoom);
+		int scaledHeight = DPIUtil.scaleUp(DPIUtil.scaleDown( imageList.height,imageList.zoom), zoom);
```

Comment:
whitespace seems to be broken here, e.g.
`DPIUtil.scaleDown( imageList.width,imageList.zoom)` -> `DPIUtil.scaleDown(imageList.width, imageList.zoom)`

Reply:
Yes, I adapted that in the latest commit